### DISCUSSION TITLE
can i use General for have contributors?

### DISCUSSION THREAD

**Discussion by jossse69** [community/community#61245] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

(this is repost cuz the og post has not got views)

**Comment by roma-marshall** [comment]
hello @jossse69, yes, it will work. Every time you create a discussion, it will show up in your github activity history as a contribution.

**Comment by mehulrajdev** [comment]
Yes. You can easily discuss and after commenting on the discussion you can see yourself as a contributor
